src/app/components/Item/ItemTableRow.jsx

```diff
@@ -3,59 +3,115 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
+import {
+  trackDiscovery,
+  ajaxCall,
+} from '../../utils/utils';
+import Actions from '@Actions';
+
 import appConfig from '../../data/appConfig';
 
-const ItemTableRow = ({ item, bibId, getRecord, searchKeywords }) => {
-  if (_isEmpty(item)) {
-    return null;
+class ItemTableRow extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.getItemRecord = this.getItemRecord.bind(this);
   }
 
-  if (item.isElectronicResource) {
-    return null;
+  getItemRecord(e) {
```

Comment:
Why do you want to redefine this function instead of passing it down? Not that the other way was better, but it might be easier now to have a utility file where you can extract this function and pass down values you need to change. Mostly saying this because it seems you have some copy/paste issues in this function like on line 29 `trackDiscovery('Item Request', 'Search Results');`, line 47 `'Error attemping to make an ajax request to fetch an item in ResultsList',`.